README.org

```diff
@@ -137,6 +138,11 @@
       If the file exists in persistent storage, it will be bind
       mounted to the target path; otherwise it will be symlinked.
 
+    - ~hideMounts~ allows you to specify whether to hide the
```

Comment:
Instead of an option that only affects GUI file managers, maybe it should instead be possible to modify the mount options wholesale?

Reply:
That would change the scope of this PR, but yes, an option to specify mount options seems more useful

Reply:
I think Cole is asking whether there is a mount option that applies to things other than GUI file managers. So that, for example, when I do `lsblk` I don't see all the bind mounts.

I don't think such an option exists though.

Reply:
No, I was asking to make this more general :P

Reply:
Would it be fine if there was a `hideMounts` option as well as another option to specify additional mount options?

Reply:
I don't particularly care which way this goes (mostly because I don't actually use this (........yet.......)), to be honest. I just figured it'd be nicer if it was more generic instead of hardcoding support for something that "only" is noticeable in GUI file managers.

So, aside from this, I think this looks good to me :)

Reply:
Yep, understood, the main reason I wanted to keep the option is because `x-gvfs-hide` is not an obvious mount flag and it would be much more convenient to include it as a toggleable option (but it would also be fine with a generic mounts option with some documentation on useful ones)